system/lib/libc/musl/src/locale/newlocale.c

```diff
@@ -7,7 +7,18 @@ locale_t newlocale(int mask, const char *name, locale_t base)
 {
 	if (*name && strcmp(name, "C") && strcmp(name, "POSIX"))
 		return 0;
-	if (!base) base = calloc(1, sizeof *base);
+	if (!base) {
+		// XXX EMSCRIPTEN: avoid a malloc (which does time(), sysconf(), sbrk(), etc.) during startup
+		static struct __locale_struct first;
+		static int used_first = 0;
+		if (!used_first) {
+			used_first = 1;
+			memset(&first, 0, sizeof first);
```

Comment:
In C, static structs are defined to be filled with the bit pattern zero, so the memset should be redundant here?